parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetInputFormat.java

```diff
@@ -230,6 +233,22 @@ public static Filter getFilter(Configuration conf) {
     return FilterCompat.get(getFilterPredicate(conf), getUnboundRecordFilterInstance(conf));
   }
 
+  /**
+   * Enables using incorrect min and max that were written to the file using a
+   * signed byte array comparison. This option is safe if all of the strings in
+   * the file contain only ASCII characters, 0-127.
+   */
+  public static void enableSignedStringMinMax(Configuration conf) {
+    conf.setBoolean(SIGNED_ORDER_ENABLED, true);
+  }
+
+  /**
+   * Returns whether using signed string min and max is enabled.
+   */
+  public static boolean isSignedStringMinMaxEnabled(Configuration conf) {
+    return conf.getBoolean(SIGNED_ORDER_ENABLED, SIGNED_ORDER_ENABLED_DEFAULT);
+  }
```

Comment:
Hey, Sorry,
I know I am the one who asked for those methods here but as @isnotinvain mentioned on https://github.com/apache/parquet-mr/pull/362 we don't want to pollute the api with too many settings that are of limited use.
How about we removing these methods from there and use the string in the test like you had before?